github/enterprise_manage_ghes.go

```diff
@@ -0,0 +1,163 @@
+// Copyright 2013 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+)
+
+// NodeQueryOptions specifies the optional parameters to the EnterpriseService
+// Node management APIS.
+type NodeQueryOptions struct {
+	// UUID filters issues based on the node UUID.
+	UUID string `url:"uuid,omitempty"`
+
+	// ClusterRoles filters The cluster roles from the cluster configuration file.
+	ClusterRoles string `url:"cluster_roles,omitempty"`
+}
+
+// ClusterStatus represents a response from the GetClusterStatus and  GetReplicationStatus method.
+type ClusterStatus struct {
+	Status *string               `json:"status"`
```

Comment:
Anywhere that you don't use `omitempty` (typically for required fields), chances are good that you don't need a reference type.
So this would most likely be "*string" => "string" if this is a required field.